src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="search-bar search-bar--big" data-qa="big">
+      <div class="search-bar__icon search-bar__icon--big"></div>
```

Comment:
This div is redundant, so you need to add the icon for the input
```suggestion
```

Reply:
i tried to change div to icon, but:
The tag <i> is banned and should not be usedlinthtml(tag-bans)